lib/Elevate/Blockers/WHM.pm

```diff
@@ -62,6 +64,44 @@ sub _blocker_is_newer_than_lts ($self) {
     return 0;
 }
 
+sub _blocker_cpanel_needs_license ($self) {
```

Comment:
maybe consider adding some unit tests for that new function?

Reply:
Considering it. I don't want to drill down too far into cPanel internals though.